src/features/processPicker.ts

```diff
@@ -389,33 +390,57 @@ function execChildProcess(process: string, workingDirectory: string): Promise<st
     });
 }
 
+// VSCode cannot find the path "c:\windows\system32\bash.exe" as bash.exe is only available on 64bit OS. 
+// It can be invoked from "c:\windows\sysnative\bash.exe", so adding "c:\windows\sysnative" to path if we identify
+// VSCode is running in windows and doesn't have it in the path.
+function GetSysNativePathIfNeeded() : Promise<Object> {
+    return PlatformInformation.GetCurrent().then(platformInfo => { 
+        if (platformInfo.isWindows && platformInfo.architecture === "x86_64") {
+            let currentPath : String = process.env.PATH;
+            let windir : String = process.env.WINDIR;
+            let sysnative : String = windir.toLowerCase() + "\\sysnative";
+            let paths : String[] = currentPath.split(";");
+            for (let path of paths) {
+                if (path.toLowerCase() === sysnative) {
+                    return {'PATH' : currentPath };
+                }
+            }
+        
+            return {'PATH' : currentPath + ";" + sysnative };            
```

Comment:
Does node merge in the current environment block? Or will this result in an environment block that only defines 'PATH' and nothing else? Maybe try launching VS Code from a command prompt with an extra environment variable and then launch `c:\windows\syswow64\cmd.exe /c set` to see if it handles this case correctly.